README.md

```diff
@@ -131,6 +135,8 @@ export class MyContainer extends Container {
   // Lifecycle method called when container shuts down
   override onStop(): void {
     console.log('Container stopped!');
+    // you can also call startAndWaitForPorts() again
```

Comment:
Any specific use case where we want to recommend this? - Feels like it'd make sense to recommend onError but not here?

Reply:
idk sometimes you want to restart a container?